migrations/20240722123726_add_sms_mfa_config.up.sql

```diff
@@ -0,0 +1,13 @@
+do $$ begin
+    alter type factor_type add value 'sms';
+exception
+    when duplicate_object then null;
+end $$;
```

Comment:
Has this exception been tested?

Reply:
Yup - delete the migration, restart auth, check that the enum is still thre

```
SELECT e.enumlabel AS value
FROM pg_type t 
JOIN pg_enum e ON t.oid = e.enumtypid 
JOIN pg_namespace n ON n.oid = t.typnamespace
WHERE t.typname = 'factor_type' AND n.nspname = 'auth';
```